lightning/src/routing/router.rs

```diff
@@ -72,21 +72,22 @@ pub struct Route {
 }
 
 impl Route {
-	/// Returns the total amount of fees paid on this Route.
-	/// This doesn't include any extra payment made to the recipient,
-	/// which can happen in excess of the amount passed to `get_route`'s `final_value_msat`.
+	/// Returns the total amount of fees paid on this [`Route`].
+	///
+	/// This doesn't include any extra payment made to the recipient, which can happen in excess of
+	/// the amount passed to [``get_route``]'s `final_value_msat`.
 	pub fn get_total_fees(&self) -> u64 {
 		// Do not count last hop of each path since that's the full value of the payment
 		return self.paths.iter()
-			.flat_map(|path| path.split_last().unwrap().1)
+			.flat_map(|path| path.split_last().map(|path_pfx| path_pfx.1).unwrap_or(&[]))
```

Comment:
This may be more readable if you destructure the tuple as `(_, path_prefix)` given only the last part of the tuple is the prefix, assuming `pfx` stands for prefix. Wasn't sure.